### ISSUE TITLE
Unraid app crashing

### ISSUE THREAD

**Issue by edalcin** [njic/medassist#5]
Just to let you know that Unraid app is crashing. However, is a nice app. I'll follow closely.

**Comment by njic** [comment]
Thanks for feedback. I'm not using Unraid, but more info could be helpful. Is it crashing on startup, are you getting anything in logs?

**Comment by edalcin** [comment]
Thanks for your feedback.

Unraid app is just a "docker" with a config interface (variable, path etc...). It runs and the log is this:

`
Server is running on http://localhost:3111
Tenadren added successfully!
Settings saved successfully!
Tenadren updated successfully!
Glifage XR 500 added successfully!
`
Then, another click on the interface and it freezes.

The "top" on the console shows this:

PID USER      PR  NI    VIRT    RES    SHR S  **%CPU**  %MEM     TIME+ COMMAND
1 root      20   0  774284  38012  19960 R  **99.3**   0.1   4:04.18 **node**
18 root      20   0    2388    764    692 S   0.0   0.0   0:00.01 sh
25 root      20   0    8052   3188   2736 R   0.0   0.0   0:00.00 top


Looks like "node" is whooping the processor and freezing everything.

I hope it helps. Is a nice tool!

**Comment by njic** [comment]
Hey dude. Please try out new release

https://github.com/njic/medassist/releases/tag/v0.15.0.develop

I fixed multiple issues here. If it turns  out it is stable enough, I'll merge it to main branch and create latest release.

I also moved database path (to achieve uniform path no matter what installation method was chosen), so make sure to update docker-compose

```
    volumes:
      - /path/to/database/directory:/app/medassist
```
      
 Change to:
 
```
     volumes:
      - /path/to/database/directory:/app/database
```
 
 Also change version tag to `develop` or `v0.15.0.develop`
 
 To backup your medication, just save `medication.db` somewhere
 
Backend script shouldn't get stuck anymore, I hope I fixed all CPU/Memory issues with this. Part of the code responsible for "Upcoming Schedules" is separated from main function. On top of that, main function doesn't count more than 365 days (if that many medication is left) and it reports with Days Left "365+" and Order Before "1+ Year"
 

 Please confirm is it working for you so I can close the issue

**Comment by edalcin** [comment]
Thanks. I'll wait for the Unraid app docker update.

**Comment by njic** [comment]
Ok, that's probably at the same time when I push this version with latest tag. Expect that in next few days

**Comment by njic** [comment]
"latest" tag has been released together with v0.16.2. I'm closing the issue since I believe its fixed, but feel free to reopen it or let me know if you face the same problem again. I'm fairly confident, but I'm looking forward to your feedback :D

**Comment by edalcin** [comment]
It's working perfectly now! Thanks for the update.
